### ISSUE TITLE
[WIN] Communication Layer (IsoTP)

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#39]
## User story
1. As a developer
2. I need a class to send and receive IsoTP messages
3. So that I can send messages bigger than normal 8 bytes length

## Acceptance criteria
* Reassemble packets on reception
* Demonstrating by sending example packets (one short and one long) on CAN bus
* Demonstrate reception of messages sent (e.g. by second process on same virtual can bus)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
